doc/Analytics.xml

```diff
@@ -640,31 +640,6 @@
         <para>When the receiver of a scene description receives an empty frame, the receiver should assume that all subsequent frames are empty as well until the next non-empty frame is received. When the last received frame is non-empty, the receiver should assume that a description of the next processed frame will be transmitted.</para>
         <section xml:id="_Ref208712481">
           <title>Objects</title>
-          <para>Objects are identified via their ObjectId. Features relating to one particular
```

Comment:
Why do you have to delete this section? in addition to ObjectId it also explains how objects are formatted. Can we not update this section with the new UUID information ?